java/arcs/android/common/resurrection/DbHelper.kt

```diff
@@ -193,23 +208,26 @@ class DbHelper(
                     component_package TEXT NOT NULL,
                     component_class TEXT NOT NULL,
                     component_type TEXT NOT NULL,
+                    notifier_id TEXT NOT NULL,
                     intent_action TEXT,
                     intent_extras BLOB,
-                    PRIMARY KEY (component_package, component_class)
+                    PRIMARY KEY (component_package, component_class, notifier_id)
                 )
             """.trimIndent(),
             """
                 CREATE TABLE requested_notifiers (
                     component_package TEXT NOT NULL,
                     component_class TEXT NOT NULL,
+                    notifier_id TEXT NOT NULL,
                     notification_key TEXT NOT NULL
                 )
             """.trimIndent(),
             """
-                CREATE INDEX notifiers_by_component 
+                CREATE INDEX notifiers_by_component_and_id
                 ON requested_notifiers (
                     component_package, 
-                    component_class
+                    component_class,
+                    notifier_id
```

Comment:
Since the Resurrection DB already exists on-device for several people, I'd recommend adding a migration step in `onUpgrade`, and bumping the `RESURRECTION_DB_VERSION`.

Reply:
Just made it drop the old tables and create new ones. SQlite doesn't let you ALTER TABLE and add primary keys. Also, there's not really any people with data that's being used, as ArcHost doesn't have resurrection service hooked up.